cmd/orb.go

```diff
@@ -506,3 +527,68 @@ func showSource(opts orbOptions) error {
 	opts.log.Info(source)
 	return nil
 }
+
+func orbInfo(opts orbOptions) error {
+	ref := opts.args[0]
+
+	info, err := api.OrbInfo(context.Background(), opts.log, opts.cl, ref)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to get info for '%s'", ref)
+	}
+
+	opts.log.Info("\n")
+
+	revisions := info.OrbVersion.Orb.Versions
+
+	if len(revisions) > 0 {
+		opts.log.Infof("Latest: %s@%s", info.OrbVersion.Orb.Name, revisions[0].Version)
+		opts.log.Infof("Last-updated: %s", revisions[0].CreatedAt)
+		opts.log.Infof("Created: %s", info.OrbVersion.Orb.CreatedAt)
+		firstRelease := revisions[len(revisions)-1]
+		opts.log.Infof("First-release: %s @ %s", firstRelease.Version, firstRelease.CreatedAt)
+
+		opts.log.Infof("Total-revisions: %d", len(revisions))
+	} else {
+		opts.log.Infof("This orb hasn't published any versions yet.")
+	}
+
+	opts.log.Info("\n")
+
+	var (
+		jobs      = 0
+		commands  = 0
+		executors = 0
+	)
+
+	var raw map[string]interface{}
+	if err := yaml.Unmarshal([]byte(info.OrbVersion.Source), &raw); err != nil {
```

Comment:
What's the motivation for using this decoding scheme for orb source versus the one in `api.go`?